src/web/modules/services/services.http.ts

```diff
@@ -0,0 +1,109 @@
+import { Request, Response } from 'express'
+import {
+  IsNotEmpty,
+  IsNumber
+} from 'class-validator'
+
+import * as logger from '../../../lib/logger'
+import { AdminUsers } from '../../../lib/pay-request'
+import { Service } from '../../../lib/pay-request/api_utils/adminUsersTypes'
```

Comment:
Nice! This interface could be under `pay-request/@types/adminUsers`, we shouldn't need to do validation on data from micro services but having the shape of the data defined is great.